src/sort.sig

```diff
@@ -1,7 +1,16 @@
+(* sorts are the classifiers of various ABTs. They're similar to types but
```

Comment:
- [ ] remove `various`
- [ ] we could say, "They're analogous to 'base' types, like `int` or `nat` or `string`."